MmSupervisorPkg/Library/DxeMmSupervisorVersionPublicationLib/DxeMmSupervisorVersionPublicationLib.c

```diff
@@ -346,6 +344,66 @@ PublishUefiVariable (
   }
 }
 
+/**
+ A protocol notification function that is invoked when the MM Supervisor Communication protocol is installed.
+
+  @param[in] Event           The signaled event.
+  @param[in] Context         Not used.
+
+**/
+VOID
+EFIAPI
+MmSupervisorCommunicationAvailable (
+  IN EFI_EVENT  Event,
+  IN VOID       *Context
+  )
+{
+  EFI_STATUS  Status;
+
+  if (mSupvCommunicationProtocol == NULL) {
+    Status = gBS->LocateProtocol (&gMmSupervisorCommunicationProtocolGuid, NULL, (VOID **)&mSupvCommunicationProtocol);
+    if (EFI_ERROR (Status)) {
+      return;
+    } else {
+      gBS->CloseEvent (Event);
+    }
+
+    if ((mSupvCommunicationProtocol != NULL) && (mVariablePolicyProtocol != NULL)) {
+      PublishUefiVariable ();
+    }
+  }
+}
+
+/**
+ A protocol notification function that is invoked when the MM Supervisor Communication protocol is installed.
+
+  @param[in] Event           The signaled event.
+  @param[in] Context         Not used.
+
+**/
+VOID
+EFIAPI
+UefiVariablePolicyAvailable (

```

Comment:
Should we add documentation to [PlatformIntegrationSteps](https://github.com/microsoft/mu_feature_mm_supv/blob/main/MmSupervisorPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md) clearly stating that a variable policy implementation must exist to publish the mm supervisor version variable?

Reply:
Yes, that would be good to call out. I'll update the steps.

Reply:
Done